source/manual/howto-backup-and-restore-in-aws-rds.html.md

```diff
@@ -224,6 +247,12 @@ sudo psql -U aws_db_admin -h local-links-manager-postgres -d local-links-manager
 
 This requires updating the CNAME for `local-links-manager-postgres`.
 
+You will need to switch your AWS role to `poweruser` apply the changes.
```

Comment:
This seems very strange — `admin` is supposed to be a superset of `poweruser` so this step shouldn't be necessary, at least in theory. Did it fail when you used your admin role and if so, do you still have the error?

Reply:
Asking cos if this really was necessary to complete the procedure then we probably have a misconfiguration somewhere (or I'm missing something 😅)

Reply:
Ah, this is me not remembering the available privileges - we originally tried `readonly` and when that didn't work tried `poweruser` 😅 
I just logged in and tested it with `admin` and that worked 👍 I've updated the doc now.

Reply:
Ahah cool no worries! Thanks for checking that. I think in that case you're probably best off removing that line altogether then, because someone following the playbook will already have admin creds at that point in the instructions (so if anything it's a little confusing to randomly call it out on that particular step).